src/components/stats/index.tsx

```diff
@@ -368,57 +368,57 @@ const TopTradingPair = ({ mode }: { mode: Mode }) => {
           </div>
           {isLoading
             ? Array(10)
-                .fill(null)
-                .map((_, idx) => <RowSkeleton key={idx} />)
+              .fill(null)
```

Comment:
Note for reviewer: 
Lines starting from this are whitespace changes. Can ignore